wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationService.swift

```diff
@@ -88,6 +88,22 @@ public final class ConversationService: ConversationServiceInterface {
         }
     }
 
+    public func createOneToOneConversation(
```

Comment:
It would be nice to format consistently with the rest of the file.

Reply:
This is intended only for team one to one conversations right?

Reply:
- I think it should renamed to `createTeamOneToOneConversation` to avoid possible confusion.
- I think the indentation of the method body has 4 spaces too many.

Reply:
Done